### DISCUSSION TITLE
LDAP integration

### DISCUSSION THREAD

**Discussion by dkoci** [noobaa/noobaa-core#6839] [Q&A]
Hi.
Is it possible to use LDAP or Keycloak for authentication in mgmt console? Is it possible to login with openshift user accounts?

**Comment by nimrod-becker** [comment]
It is possible to use OAUTH with openshift accounts.
It is not possible to use other providers for authenticating into the console.

**Reply by dkoci** [reply]
`It is possible to use OAUTH with openshift accounts` this would be ok. I need some tips how to set it up. Is there any documentation available?
